sklearn/base.py

```diff
@@ -52,8 +52,8 @@ def clone(estimator, *, safe=True):
 
     Parameters
     ----------
-    estimator : estimator object, or list, tuple or set of objects
-        The estimator or group of estimators to be cloned
+    estimator : {list, tuple, set} of estimator object or estimator object
```

Comment:
Nit:

```suggestion
    estimator : {list, tuple, set} of estimator objects or estimator object
```

Reply:
Done. Thank you very much for your review!